flax/linen/recurrent.py

```diff
@@ -497,3 +500,189 @@ def initialize_carry(rng, batch_dims, size, init_fn=zeros):
     key1, key2 = random.split(rng)
     mem_shape = batch_dims + size
     return init_fn(key1, mem_shape), init_fn(key2, mem_shape)
+
+class RNNBase(Module):
+  """Base class for RNN layers, in contains an `rnn_forward` method
+  that runs the scan loop.
+  """
+  def rnn_forward(
+    self,
+    cell: RNNCellBase,
+    x: jnp.ndarray,
+    *,
+    init_key: jnp.ndarray,
+    time_axis: int,
+    num_feature_dims: int,
+    stateful: bool,
+    return_sequences: bool,
+    return_state: bool,
+    go_backwards: bool,
```

Comment:
I would just use the same term (reverse) here as lax.scan

Reply:
Done.